akka-actor/src/main/scala-2.13-/akka/util/ccompat/CompatImpl.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.util.ccompat
+
+import scala.collection.generic.CanBuildFrom
+import scala.collection.mutable.Builder
+
+/**
+ * Based on https://github.com/scala/scala-collection-compat/blob/master/compat/src/main/scala-2.11_2.12/scala/collection/compat/CompatImpl.scala
+ */
+private[ccompat] object CompatImpl {
+  def simpleCBF[A, C](f: ⇒ Builder[A, C]): CanBuildFrom[Any, A, C] = new CanBuildFrom[Any, A, C] {
```

Comment:
This is only used in one place, so maybe just move this method there and mark it private? (not package private)

Reply:
I wanted to keep the structure similar to collection-compat to make it easier to port updates later

Reply:
Hmm... I see _thinking about pros and cons_

(I think you apply it inconsistently though with, for example, `ImmutableSortedMapExtensions` as a distinct file.)

Reply:
Yeah, I tried putting `ImmutableSortedMapExtensions` inside `PackageShared.scala` but that wouldn't compile :/ (something something package object something)

Reply:
Ah, I hate when I get that error message. It's so unspecific! (:trollface:)